lib/config/presets/local/common.ts

```diff
@@ -17,10 +17,7 @@ export async function fetchJSONFile(
       throw err;
     }
 
-    logger.debug(
-      { err, repo, fileName },
-      `Failed to retrieve ${fileName} from repo ${repo}`
-    );
+    logger.debug(`Could not fetch ${fileName} from repo ${repo}`);
```

Comment:
why not same message as the others?
```suggestion
    logger.debug(`Preset file ${fileName} not found in ${repo}`);
```